src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationValidService.java

```diff
@@ -29,6 +30,8 @@ boolean isFallbackLetterRequiredForSubscriptionType(NotificationWrapper wrapper,
             || APPOINTEE.equals(subscriptionType)
             || (REPRESENTATIVE.equals(subscriptionType) && null != wrapper.getNewSscsCaseData().getAppeal().getRep()))) {
             result = true;
+        } else if (DWP_RESPONSE_RECEIVED_NOTIFICATION.equals(eventType)) {
```

Comment:
DWP_RESPONSE_RECEIVED_NOTIFICATION should be added to the FALLBACK_LETTER_SUBSCRIPTION_TYPES list.  Then there will be no need for this extra check.